docs/salesforce/release-notes.mdx

```diff
@@ -21,6 +21,26 @@ import {
 
 # Release Notes
 
+## 3.10
+
+_May 31, 2024_
+
+** Release Notes **
+
+- Salesforce integration will support LanguageAI, which allows agents to converse with multi-lingual end users.
+- Salesforce messenger will support 47 languages out of the box, with English as default. Details of the supported languages can be found [here](https://developers.helpshift.com/android/i18n/).
+- Agents can now reply to customers in their languages and translate customer replies to English.
+- Bug fixes
+
+<Admonition type="info" title="Note">
```

Comment:
Do we need to add this note? Since we have very few customers, we can just reach out to them directly.

Reply:
ok